---
Title: Impact and influence of the natural *Vibrio*-squid symbiosis in understanding bacterial-animal interactions  
Date: \today  
...

# Abstract 

Animals are colonized by bacteria and in many cases partners have co-evolved to perform mutually beneficial functions. An exciting and ongoing legacy of the past decade has been an expansion of technology to enable study of natural associations *in situ* (*in vivo*). As a result more symbioses are being examined, and additional details are being revealed for well-studied systems. In this context we review literature from the *Vibrio fischeri*-*Euprymna scolopes* symbiosis with a focus on key studies that have had an impact on understanding bacteria-animal interactions broadly. This is not intended to be a comprehensive review of the system, but rather to focus on particular studies, discuss the key findings of the work, and to highlight connections with other symbiotic systems. Here we discuss how bacteria influence animal development; how microbial developmental biology regulates developmental transitions; the nature of molecular dialogue between bacteria and their animal hosts; how small genetic changes can have large effects on host colonization specificity; and how bacterial metabolism is constrained by the host environment and life cycle.

# Main Text

Rough outline of this introductory section:
- Microbiome has gotten big! 1 paragraph on microbiomes in humans ("microbial organ"), animals, (and plants...but not focus here).
- Introduce the V. fischeri-squid system.
- Focus of this review is the processes by which the initial microbial population is selectively acquired, influences host biology, and is stably maintained. Focusing on a single influential model system, five key findings from the past 25 years are presented to demonstrate how they impacted the study system and our impact broadly of the relationships of bacteria and their animal hosts.

## Isolating the colonization process in a natural model system

Animals born and then harvest symbionts from the environment
- could discuss vertical/horizontal transmission.

**What makes a model valuable?** Why has the *V. fischeri*-squid model provided valuable insights over a long period?
- reductionist
- genetically-tractable bacterium
- ability to study individual hosts, especially for imaging
- robust natural system -- all animals colonized in 3 hours; enables...
- synchronous colonization. Has been key to understanding developmental processes (aggregation; 12 h signal; etc)

## Shedding the cell wall to say "I'm here"

A particularly dramatic example of a role for bacteria influencing a specific host developmental process was reported by Montgomery and McFall-Ngai [@montgomery1994], when they reported that in response to symbiotic colonization by *V. fischeri* the ciliated appendages of the *E. scolopes* light organ undergo regression during the subsequent five days. The bacterial peptidoglycan fragment, tracheal cytotoxin (TCT), which previously had been shown to induce apoptosis in ciliated epithelium upon release from *Bordetella pertusis*, was identified to perform a similar function in *V. fischeri*, but this time with a resulting beneficial outcome [@koropatnick2004]. The Lipid A portion of lipopolysaccharide (LPS) is required to recapitulate the apoptosis observed from intact *V. fischeri*. The cell death from these compounds, in conjunction with hemocyte trafficking that is also induced from TCT, results in the regression phentoype. <!-- NEED TO INCLUDE INFO ON LIGHT -->

<!-- TCT not resorbed in the cell? Is this published? -->

<!-- TCT in specificity? Do similar Vibrios release TCT? Does it work? -->

TCT plays a key role in signaling to multiple host phenotypes, is sufficient to recapitulate many of the above phenotypes (although not always to the full effect), and likely contributes to the specificity of the host responses observed upon *V. fischeri* colonization. However, there are other bacterial and host compounds that have been characterized for their role in signaling and these are summarized below.

This work in *V. fischeri* was influenced by studies in invertebrate systems that demonstrated host development in response to symbiont colonization and vertebrates that showed general responses to consortia <!-- specific studied and general --> reviewed in [@montgomery1994]. Following up on this work to define specific compounds that influenced host development, studies in other systems began to look critically at the molecular interface between microbes and host. In 2001 it was shown in a mammalina system that in response to colonization by gut *Bacteroidetes* such as *Bacteroides thetaiotaomicron*, terminal tissue differentiation (e.g., fucosylation) is dependent on the presence of the symbiotic bacteria [@hooper2001a].  <!-- zebrafish? -->


## Light up my life

Needs a lead in….maybe about sanctioning and selection in other host-microbe relationships?  

In 2000, Visick et al. reported a key role for light production in the development of the squid-vibrio symbiosis and persistence of the bacterial symbiont.  Specifically, this study demonstrated a direct role of luciferase (the enzyme that produces light) in the induction of host development and bacterial symbiotic persistence (Visick 2000).  Prior to this report, although it was known that bacteria produced light during the symbiosis, the significance of bioluminescence in the symbiosis was not clear.    By testing colonization ability of strains with mutations either in a structural luminescence gene or luminescence regulatory genes, it was discovered that these mutant strains colonized juvenile squid to the same levels as wild type in the first 24 hours.  However, by 48 hours there was a three- to four-fold reduction in colonization by the dark mutants as compared to wild type.  In addition, levels of the mutant strains also decreased in squid co-colonized with a luminescence mutant and wild type, indicating the presence of light-producing wild-type cells in the light organ could not complement the colonization defect of the light-deficient cells.  These results suggested that the ability for the bacteria to produce light was important for persistence in the light organ and that somehow the non-luminescent cells are selected against during development of the symbiosis.  

Interestingly, the light-deficient strains had a specific effect on host development.  Although colonization by a luminescence mutant still triggered apoptosis-related developmental changes in the ciliated surface of the light organ, colonization of the tissue by these strains no longer increased cell swelling of the epithelial cells lining the light organ crypt spaces.  Therefore, light production appeared to play a specific role in host developmental pathways.  Notably, this was the first report of V. fischeri genes required for induction of bacterial-triggered differentiation of host tissue.  

It was later discovered that the antibiotic markers and method for constructing the mutants used by Visick et al. (visick 2000) resulted in colonization attenuation and pleotropic effects.  To further confirm the role of light production in symbiotic colonization, Bose et al. (Bose 2008) took advantage of newly developed tools to construct strains lacking the luminescence structural genes, as well as strains where expression of the luminescence structural genes was uncoupled from normal cell regulatory pathways.  Using these strains, the results of Visick et al were confirmed, demonstrating that the strain lacking the luminescence structural genes displayed a four-fold reduction in colonization as compared to wild type.  However, these studies were limited to the early stages of colonization due to the difficulty of raising juvenile animals to adulthood.  Previous studies had suggested that maintenance of the symbiosis over the life of the animal requires a maturation process of several weeks (Montgomery and McFall-Ngai 1998; PMID 9924775), leaving the question of how production of light influences symbiosis maturation beyond 72 hours.  

A major breakthrough for laboratories studying the system came with the development of modified protocols for simplified rearing of newly-hatched juvenile squid through and beyond the maturation process.  These methods allowed investigation of how bacterial-produced light affects the development of the symbiosis over four weeks (Koch 2013).  In these studies, the levels of the luminescence-deficient mutant associated with the squid light organ continued to decrease over time, reaching the level of detection after 28 days.  Similar results were observed in squid colonized with mixed inocula containing both wild type and the luminescence mutant, where after 15 days the mutant was barely detected.  Therefore the decreased persistence observed during early colonization becomes more pronounced as the symbiosis matures, with eventual loss (or near loss) of non-luminescent strains.  

One of the most interesting results of this work involved experiments to address the influence of a previous colonization event on recolonization by wild type.  For these experiments, juveniles were colonized with either wild type or the luminescence mutant.  After one, three, or five days the animals were treated with antibiotics to clear the bacteria from the light organ and then exposed again to wild-type V. fischeri.  This experimental design allows for testing whether particular strains "signal" to the host their suitability in terms of symbiotic light production and if this influences the maturation of the symbiotic state.  If wild-type animals were cured after one day, they were readily recolonized.  However, this was not the case if cured after five days.  In these animals a refractory state was induced by the wild-type bacteria that prevented recolonization.  In contrast, after five days greater than 80% of the animals previously colonized by the luminescence mutant were recolonized by wild type.  These results support the idea that the host is detecting light production by bacterial cells and/or is altering physiological conditions to sanction the non-luminescent strains.  In addition, the host apparently is able to "eject" an inappropriate light deficient strain, while allowing future recolonization by a symbiotically appropriate light-producing strain.     

These studies bring up interesting questions about how the animal senses the light and/or dark bacteria, how the dark strains are selected against, and the nature of the mechanism used to allow recolonization of light organs by luminescent strains after curing of strains unable to produce light.  Answers to these questions are actively being pursued, and towards the question of light sensing by the host, several recent studies have identified host cryptochrome and eye-specification genes whose expression appears to be influenced by the light produced by bacteria (Peyer et al 2014; PMID 24157521; Heath-Heckman 2013; PMID 23549919). However, it is not currently clear how this might be related to selection of light-producing strains.  Although light production is not a general aspect of all host-microbe interactions, deciphering the mechanisms hosts use to select microbial partners in diverse systems will be invaluable for increasing our general understanding of the requirements for effective initiation and maintenance of horizontally-transmitted symbioses.  


## NO way in

There is a long history of the study of NO in eukaryotes, and this small diffusible molecule has been implicated in many different cellular processes including signaling and innate immunity (Fang 2004, PMID 15378046).   Although the roles for NO in eukaryotic physiology and defense against pathogens was discovered many years ago, through the study of the squid-vibrio and other symbioses (Damiani 2016; PMID 27092165) it has become apparent that NO also influences the establishment and maintenance of mutualistic host-microbe relationships as both a signal and a specificity determinant (Wang and Ruby 2010; 21338463).  

A key study revealing a role for NO in animal-microbe symbioses was published by Davidson, et al. (2004).  In this work the authors demonstrated that during the initiation of colonization in the squid-vibrio system, NO is produced in host tissue through the activity of nitric oxide synthase (NOS), and this activity was attenuated after successful colonization by V. fischeri.  Using staining and immunocytochemistry NOS and NO were found located in the epithelium of the light organ, as well as in vesicles within mucus shed from these cells.  It is within this mucus that the bacterial cells aggregate prior to entering the light organ.  Treatment of the animals with a NO scavenger allowed large aggregates of non-symbiotic vibrios to form, but these bacteria did not go on to colonize.   However, in the absence of a NO scavenger V. fischeri aggregated in the mucus, went on to colonize, and after colonization NOS activity and NO production were attenuated.  The results strongly suggested that NO acts as a specificity determinant, helping to limit aggregation of non-symbiotic vibrios and select for symbiotically competent V. fischeri from the mixed microbial population found in seawater.  Of course these results also suggested that V. fischeri would need to effectively sense and respond to the host-produced NO to successfully establish the partnership.  

Since this initial report, progress has been made in determining how V. fischeri senses and responds to NO.  In 2010, Wang et al. described a role for the NO-detoxifying enzyme flavohenoglobin (Hmp) (poole and hughes, 2000; PMID 10844666) in the protection of V. fischeri from NO during host colonization.  Using genetic approaches it was demonstrated that a strain lacking hmp displayed a colonization deficiency.  Although it had been previously predicted using bioinformatic analyses (Rodionov, 2005; PMID 16261196), in this study expression of hmp was linked to the activity of a NO-responsive negative regulator NsrR ( Tucker, 2010; PMID20167493—ref about NsrR).  However, NsrR is not the only important NO-sensing regulator in V. fischeri.  H-NOX, a heme NO/oxygen-binding protein also plays a role in symbiotically relevant NO-responsive regulation of genes in V. fischeri (Wang 2010).  Although H-NOX-like proteins are widely distributed in bacteria, this was the first report describing bacterial H-NOX function.   Interestingly, it appears that one role for H-NOX in V. fischeri is to sense NO and correspondingly suppress bacterial hemin uptake during the early stages of host colonization.  The authors predicted that early repression of iron uptake would protect the cells from the potentially harmful effects of Fenton chemistry while exposed to host-generated oxidants.  This repression would then be relieved upon bacterial-influenced attenuation of oxidant production (Davidson 2004), and hemin uptake genes would be expressed to support growth in the iron-limited light organ environment (Graf and Ruby 2000).   A subsequent study demonstrated that the hemin uptake genes in V. fischeri are induced during the later stages of symbiotic colonization and deletion of these genes negatively impacted colonization (Septer 2011 PMID 21883801).   Together these results demonstrate that although the ability to sense and detoxify NO is important in terms of symbiotic specificity, NO also acts as a signal to modulate bacterial gene expression to promote successful colonization.  

Paragraph about link to other systems and future work??
NO = defense against pathogens, but parallels in symbiotic systems just a different outcome.  Hmp is important defense in pathogens, Plant-microbe systems (Damiani 2016; PMID 27092165) also NO important role.    Same language, different outcome.  Future work looking more globally at response = insight.

## Swimming against the flow

Flagellar development and chemotaxis

key paper: Ruby and Asato 1993 [@ruby1993]; mechanism (sort-of): [@brennan2013a, @brennan2013, @nikolakakis2016]


## You talking to me?

A 2009 study by Mandel et al. revealed that a single regulatory locus can be critical to enable an expansion of bacterial host range into a new animal host [@mandel2009].

A comparative genomic study examined differences between two *V. fischeri* strains, the squid-colonizing strain ES114 and the fish isolate MJ11 [@mandel2009]. Most of the genomes were shared with protein-coding genes that were practically identical [XXX - fill in with numbers], but approximately 400 protein-coding genes were unique to one strain or the other. Analysis of these factors revealed that the squid biofilm regulator, RscS, was encoded in the squid symbiont but not in the fish symbiont. The known RscS target genes, *sypA, sypB, ..., sypR* were encoded in both genomes and fairly conserved (> XXX % amino acid conservation). It was known previously that mutants in strain ES114 that lacked RscS are unable to productively colonize the squid [@visick2001]. Therefore, the study asked whether the absence of the regulator was sufficient to explain the differential colonization phenotype. Introduction of RscS into strain MJ11 was sufficient to allow it to colonize the squid host. Phylogenetic analyses presented supported a model in which MJ11 was part of an ancestral group of *V. fischeri* that lacked *rscS*, and that this gene was acquired coincident with squid colonization of squid in the North Pacific Ocean (i.e., Japan and Hawaii).

A single gene that was sufficient to shift the animal hosts available to a bacterium was extreme but consistent with reports that had been showing that small loci could regulate microbe-host specificity. Work in entomopathogenic nematodes had shown that symbiotic *Xenorhabdus nematophila* requires the three-gene *nilABC* locus for colonization, and that expression of these factors in a heterologous symbiont is sufficient to enable colonization of *Steinernema carpocapsae*, the worm host that otherwise is specific for *X. nematophila* [@cowles2008].
[XXX Also describe Yersinia and Rhizobia here XXX. Probably say something in this paragraph about genomic islands and Salmonella Typhimurium / Typhi.]

In the human gut microbiome there are examples in which single gene changes have been critical. XXX
- B. fragilis PSA as specificity factor [@mazmanian2008]

Horizontal gene transfer enables bacteria to gain new functions, and the V. fischeri work demonstrated the profound impact of acquisition of a single gene. A study in Japan has identified transfer of carbohydrate-degrading enzymes from seaweed to microbiota constituents [@hehemann2010].

The work by Mandel et al. [@mandel2009] revealed a single regulatory gene that can alter host range. Questions in *V. fischeri* remain as to how this gene was able to hijack a regulatory pathway, and how the evolution of signal transduction pathways occurs as interactions between host and microbe evolve. [XXX needs some work]

[could discuss host factors, NO, hemocytes & regression that likely contribute to specificity but have not been shown to in the intact animal.]


## I don't know who you are

(The symbiosis provides a useful framework for identifying gene function)

Aox

halovibrins

(even BinK & RscS from global screens)

Maybe there are better examples...

key paper: Dunn et al. 2010 [@dunn2010]



## So that's what matters...

(The symbiosis as a focusing agent to identify relevant pathways in vivo)

Ideas:

[@wier2010] just lists of genes until modeled >> focus on glycerol and chitin. Glycerol is important in juvenile but not adults. Chitin plays many important roles...

Syp & cellulose biofilms. Both impressive in culture, both yield crystal violet phenotype, only Syp connected to host colonization.


Old ideas...

 also [@wier2010, @schwartzman2014]

aerobic/anaerobic. Day/night.

make connections to diel/circadian biology



##### Aggregation and Biofilm Formation

key paper: Nyholm et al. 2000 [@nyholm2000]; mechanism: [@visick2001, @yip2005, @yip2006]

